pkgs/applications/networking/cluster/k3s/update.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq
+
+set -eou pipefail
+
+cd $(dirname "${BASH_SOURCE[0]}")
+
+LATEST_TAG_RAWFILE=$(mktemp)
```

Comment:
Doing a `mktemp` per file we need definitely works, but I think it might be slightly simpler to keep all the working state in one dir, i.e. to do `WORKDIR=$(mktemp -d)`, and then put the remaining files in that directory.

This would also make it easier to cleanup if we care to (i.e. `trap 'rm -rf $WORKDIR' EXIT`).

Both of the above nits/suggestions are non-essential. What we have works, and still reads clearly, I just think a single directory / mktemp invocation seems slightly cleaner.

Reply:
Done